build/Cargo.toml

```diff
@@ -10,7 +10,4 @@ repository = "https://github.com/napi-rs/napi-rs"
 version = "1.0.1"
 
 [dependencies]
-cfg-if = "1"
-
-[target.'cfg(windows)'.dependencies]
```

Comment:
It may not be a common practice, but I've compiled rust into a windows exe from Linux before, the instructions at https://github.com/est31/msvc-wine-rust work very well

Reply:
Looks great! But at least we should exclude the `ureq` on Linux-musl and non-x86-x64 host. It`s not possible cross compile windows-msvc on these platform.
And from the CI, install `ureq` failed on Linux-musl host.

Reply:
Looks like the problem is with the ring dependency of rustls dep in ureq. There is a issue to reinstate native-tls support in ureq over at https://github.com/algesten/ureq/issues/319.

I updated it to exclude the windows code on musl targets. I don't think there'll be many people trying to cross compile to windows from their Raspberry Pi build server using wine and qemu, but hey there might be someone somewhere!

Reply:
Hmm there seems to be something up with the aarch64-linux action, it's not getting to the build step due to some dependency issues

```
The following packages have unmet dependencies:
 g++-9-aarch64-linux-gnu : Depends: libstdc++-9-dev-arm64-cross (>= 8) but it is not going to be installed
 gcc-9-aarch64-linux-gnu : Depends: libgcc-9-dev-arm64-cross (>= 9.3.0-17ubuntu1~20.04cross2) but it is not going to be installed
                           Recommends: libc6-dev-arm64-cross (>= 2.13-0ubuntu6) but it is not installable
```